lib/api/swagger.js

```diff
@@ -34,12 +34,7 @@ module.exports = function generateSwagger () {
   const routes = [];
 
   global.kuzzle.config.http.routes.forEach(_route => routes.push(_.assign({}, _route)));
-
-  global.kuzzle.pluginsManager.routes.forEach(_route => {
-    const route = _.assign({}, _route);
-    route.path = `/_plugin${route.path}`;
-    routes.push(route);
-  });
+  global.kuzzle.pluginsManager.routes.forEach(_route => routes.push(_.assign({}, _route)));
```

Comment:
Nitpicking: we should avoid lodash when we can
```suggestion
  global.kuzzle.pluginsManager.routes.forEach(_route => routes.push({ ..._route });
```